### ISSUE TITLE
toggle on the biometric screen does not have an accessibility label

### ISSUE THREAD

**Issue by cvarjao** [bcgov/bc-wallet-mobile#827]

**Comment by bryce-mcmath** [comment]
`Switch` components in [React Native](https://reactnative.dev/docs/0.66/switch#props) don't have support for accessibility props / attributes. Looking for a workaround
